pkg/api/plugins.go

```diff
@@ -364,6 +364,17 @@ func (hs *HTTPServer) getCachedPluginSettings(pluginID string, user *models.Sign
 	return query.Result, nil
 }
 
+func (hs *HTTPServer) GetPluginErrorsList(c *models.ReqContext) Response {
+	pluginErrors := make([]plugins.PluginErrorInfo, 0)
```

Comment:
Would move this to a function in plugins package to get the list of errors ready to use